packages/nitro-protocol/contracts/test/TestEthAssetHolder.sol

```diff
@@ -7,5 +7,7 @@ import '../ETHAssetHolder.sol';
  * @dev This contract is a clone of the ETHAssetHolder contract. It will be initialized to point to the TestNitroAdjudicator.
  */
 contract TestEthAssetHolder is ETHAssetHolder {
+    uint256 public dummy; // this is simply to make the contract have distinct bytecode from the ETHAssetHolder (otherwise bytecode verification can fail)
```

Comment:
nit: We could _really_ make sure the bytecode is unique?

```suggestion
    string public contractName = 'TestEthAssetHolder'; // ensures the contract has different bytecode from the ETHAssetHolder (otherwise bytecode verification can fail)
```